main/createUiDefinition.json

```diff
@@ -296,17 +248,11 @@
             "type": "Microsoft.Common.DropDown",
             "label": "Security group",
             "toolTip": "Select inbound ports.",
-            "multiselect": true,
-            "selectAll": true,
             "constraints": {
               "allowedValues": [
                 {
                   "label": "Bootstrap UI (443)",
                   "value": "bootstrapUi"
-                },
-                {
-                  "label": "SSH (22)",
```

Comment:
This needed for debug-cli access right?

Reply:
I removed SSH port from inbound security rules because we don't provide this functionality to the customer.